BCCCP.iml

```diff
@@ -3,9 +3,20 @@
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/test" isTestSource="true" />
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library" scope="TEST">
+      <library name="JUnit5">
+        <CLASSES>
+          <root url="jar://$MODULE_DIR$/lib/junit-jupiter-api-5.0.0-M5.jar!/" />
```

Comment:
Jupiter seems to be where the respective imports are for JUnit5 - so just a query: this version of JUnit seems to be fairly recent release and not mentioned by Jim in his lecture/how-to. Is this likely to cause any issues with him I wonder - hopefully not!